cmd/updater/diffdumps/cmd.go

```diff
@@ -78,13 +78,24 @@ func generateK8sDiffs(outputDir string, baseZipR *zip.ReadCloser, headZipR *zip.
 	baseFiles := make(map[string]*zip.File)
 	for _, baseF := range baseZipR.File {
 		name := baseF.Name
+
+		if strings.Contains(name, "..") {
```

Comment:
Shouldn't we only check for a specific component of the path being `..`? Like, a file name like "README..txt" is valid and does not cause zip slip.

Reply:
Good idea. I copied what codeql said to do, but https://snyk.io/research/zip-slip-vulnerability has a better check

Reply:
actually, going to just switch it to `../`. Idea being `../` covers all cases except when the path ends with `..`, but we check the extension so we already implicitly disallow `..` at the end